### ISSUE TITLE
Plugin crashes when logging in to VSTS - WebStorm

### ISSUE THREAD

**Issue by elliotleelewis** [microsoft/azure-devops-intellij#107]
When I click the icon in the corner of my window, then click sign in, the following happens.

1. Microsoft login page appears in which I type in my work email address
2. Page says "redirecting to organization login"
3. At my company, Okta is used for auth, so we're automatically logged into our accounts when we're logged into our machines. So the dialog closes.
4. Error popup comes up in corner of WebStorm window saying plugin has crashed.

When clicking on the error popup, the following error message is displayed:

```
getAuthenticationInfoAsync failed: KEY_VSO_AUTH_FAILED
com.microsoft.alm.plugin.exceptions.TeamServicesException: KEY_VSO_AUTH_FAILED
	at com.microsoft.alm.plugin.authentication.facades.VsoAuthInfoProvider.getAuthenticationInfoAsync(VsoAuthInfoProvider.java:98)
	at com.microsoft.alm.plugin.authentication.VsoAuthenticationProvider.authenticateAsync(VsoAuthenticationProvider.java:69)
	at com.microsoft.alm.plugin.authentication.AuthHelper.getAuthenticationInfoSynchronously(AuthHelper.java:59)
	at com.microsoft.alm.plugin.context.ServerContextManager.getAuthenticationInfo(ServerContextManager.java:489)
	at com.microsoft.alm.plugin.context.ServerContextManager.createContextFromGitRemoteUrl(ServerContextManager.java:372)
	at com.microsoft.alm.plugin.context.ServerContextManager.createContextFromGitRemoteUrl(ServerContextManager.java:353)
	at com.microsoft.alm.plugin.context.ServerContextManager.getAuthenticatedContext(ServerContextManager.java:333)
	at com.microsoft.alm.plugin.context.ServerContextManager.getUpdatedContext(ServerContextManager.java:563)
	at com.microsoft.alm.plugin.operations.Operation$1.run(Operation.java:184)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
```

**Comment by Arikael** [comment]
same happens here.
We are using ADFS for Auth. also with SSO.

I was able to enter my email address once. after that it just shows empty popups and then crashes.
I even uninstalled and re-installed the plugin but somewhere my previously entered email address is stored.

**Comment by jchharris** [comment]
I have the same issue with the plugin in Intellij after it says "redirecting to organisation login". Has anybody found a workaround?

**Comment by ForNeVeR** [comment]
Currently we believe that this is a duplicate of a fixed issue #158 (or, more precisely, #158 was a duplicate of this issue).
